src/App.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react';
+import './App.css';
+import Solution from './components/Solution';
+import Score from './components/Score';
+import Letters from './components/Letters';
+import EndGame from './components/EndGame';
+
+export class App extends Component {
+  constructor() {
+    super()
+    this.state = {
+      lettersStatus: this.generateLetterStatuses(),
+      solution: this.random(),
+      score: 100,
+    }
+  }
+
+  countDuplicates=(str)=>{
+    let check={}
+    let counter=0
+    for(let x = 0, length = str.length; x < length; x++) {
+        var l = str.charAt(x)
+        check[l] = (isNaN(check[l]) ? 1 : check[l] +1)
+    }
+    for (let key in check)
+     if(check[key]>1){
+      counter+=check[key]-1
+     } 
+    return counter 
+  }
+
+  random = () => {
+    let data = [
+      { word: 'SMELLYCAT', hint: 'what are they fiding you' },
+      { word: 'GANTZ', hint: 'An Israeli Politician' },
+      { word: 'BURGERRANCH', hint: 'An Israeli Burgers store' },
+      { word: 'BOWLING', hint: 'Enjoing Sportive throwing ball activity' },
+      { word: 'SUNGLASSES', hint: 'Coolish Summer Item' },
+    ]
```

Comment:
this should be outside of this function, in the constructor or even in separate file